app/perfaddresscache.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include "perfdwarfdiecache.h"
 
+#include "perfregisterinfo.h"
```

Comment:
exchange with the empty line above, to better group includes

Reply:
actually, remove - you also added the include to the `perfaddresscache.h` after all